<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolCollect</class>
 <widget class="QWidget" name="ToolCollect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ToolCollect</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_sidebar">
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>flechazo</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_sidebar_info">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>180</height>
         </size>
        </property>
        <property name="title">
         <string>info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_sidebar_log">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_toolcollect">
     <property name="title">
      <string>view</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_toolcollect_config">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="title">
         <string>path</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_toolcollect_config_path"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_toolcollect_config_open">
           <property name="text">
            <string>选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_toolcollect_config_flush">
           <property name="text">
            <string>读取</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_toolcollect">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>240</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="tab_toolcollect_frp">
         <attribute name="title">
          <string>frp</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_toolcollect_frp_config">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>90</height>
             </size>
            </property>
            <property name="title">
             <string>配置</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="lineEdit_toolcollect_frp_config_path">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_toolcollect_frp_config_path">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>选择frp路径</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_toolcollect_frp_config_read">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>读取配置</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_toolcollect_frp_view" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QGroupBox" name="groupBox_toolcollect_frp_view_service">
               <property name="maximumSize">
                <size>
                 <width>360</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="title">
                <string>frps</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QTabWidget" name="tabWidget_toolcollect_frp_view_service">
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <widget class="QWidget" name="tab_toolcollect_frp_view_service_config">
                   <attribute name="title">
                    <string>服务端配置</string>
                   </attribute>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_toolcollect_frp_view_service_config_port">
                      <property name="text">
                       <string>port</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_service_config_port">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>30</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_toolcollect_frp_view_service_config_token">
                      <property name="text">
                       <string>token</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_service_config_token">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>30</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0" colspan="2">
                     <widget class="QPushButton" name="pushButton_toolcollect_frp_view_config_service">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>开始服务</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="tab_toolcollect_frp_view_servicelist">
                   <attribute name="title">
                    <string>服务端列表</string>
                   </attribute>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_toolcollect_frp_view_client">
               <property name="title">
                <string>frpc</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QTabWidget" name="tabWidget_toolcollect_frp_view_client">
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <widget class="QWidget" name="tab_toolcollect_frp_view_client">
                   <attribute name="title">
                    <string>客户端配置</string>
                   </attribute>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QGroupBox" name="groupBox_toolcollect_frp_view_client_service">
                      <property name="title">
                       <string>service</string>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_2">
                       <item row="0" column="0" colspan="2">
                        <widget class="QLabel" name="label_toolcollect_frp_view_client_service_ip">
                         <property name="text">
                          <string>serviceip</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="2">
                        <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_client_service_ip">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>25</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0" colspan="2">
                        <widget class="QLabel" name="label_toolcollect_frp_view_client_service_port">
                         <property name="text">
                          <string>serviceport</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="2">
                        <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_client_service_port">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>25</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="0">
                        <widget class="QLabel" name="label_toolcollect_frp_view_client_service_token">
                         <property name="text">
                          <string>token</string>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="2">
                        <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_client_service_token">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>25</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QGroupBox" name="groupBox_toolcollect_frp_view_client_config">
                      <property name="title">
                       <string>client</string>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_3">
                       <item row="2" column="0">
                        <widget class="QLabel" name="label_toolcollect_frp_view_client_config_token">
                         <property name="text">
                          <string>token</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="2">
                        <widget class="QLabel" name="label_toolcollect_frp_view_client_config_localport">
                         <property name="text">
                          <string>localport</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLabel" name="label_toolcollect_frp_view_client_config_localip">
                         <property name="text">
                          <string>localip</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_toolcollect_frp_view_client_config_name">
                         <property name="text">
                          <string>name</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="1">
                        <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_client_config_localip">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>25</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_client_config_name">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>25</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="2">
                        <widget class="QLabel" name="label_toolcollect_frp_view_client_config_type">
                         <property name="text">
                          <string>type</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="3">
                        <widget class="QComboBox" name="comboBox_toolcollect_frp_view_client_config_type">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>25</height>
                          </size>
                         </property>
                         <item>
                          <property name="text">
                           <string>tcp</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>udp</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>http</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>https</string>
                          </property>
                         </item>
                        </widget>
                       </item>
                       <item row="2" column="1">
                        <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_client_config_token">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>25</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="3">
                        <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_client_config_localport">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>25</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="2">
                        <widget class="QLabel" name="label_toolcollect_frp_view_client_config_remoteport">
                         <property name="text">
                          <string>remoteport</string>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="3">
                        <widget class="QLineEdit" name="lineEdit_toolcollect_frp_view_client_config_remoteport">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>25</height>
                          </size>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_toolcollect_frp_view_client">
                      <property name="text">
                       <string>开始服务</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="tab_toolcollect_frp_view_clientlist">
                   <attribute name="title">
                    <string>客户端列表</string>
                   </attribute>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_toolcollect_code">
         <attribute name="title">
          <string>代码</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea_toolcollect_code">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>830</width>
               <height>464</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_toolcollect_com">
         <attribute name="title">
          <string>通信</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_toolcollect_web">
         <attribute name="title">
          <string>网站</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QGroupBox" name="groupBox_toolcollect_web">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
            <property name="title">
             <string>网站配置</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="lineEdit_toolcollect_web_path">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_toolcollect_web_path_open">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_toolcollect_web_path_read">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>读取</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_toolcollect_web" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_toolcollect_config_operate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="title">
         <string>操作</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_toolcollect_config_operate_add">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_toolcollect_config_operate_delete">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_toolcollect_config_operate_openlocal">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>打开仓库</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_toolcollect_config_operate_open">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>打开工具</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
